src/main/java/com/arpnetworking/tsdcore/sinks/WavefrontSink.java

```diff
@@ -66,9 +67,14 @@ public void recordAggregateData(final PeriodicData periodicData) {
             // Build the list of annotations based on cluster and white listed dimensions
             final Map<String, String> annotations = Maps.newHashMap();
             annotations.put("cluster", fqdsn.getCluster());
-            periodicData.getDimensions().entrySet().stream()
-                    .filter(entry -> _whiteListedDimensions.contains(entry.getKey()))
-                    .forEach(entry -> annotations.put(entry.getKey(), entry.getValue()));
+            for (final Map.Entry<String, String> dimension : periodicData.getDimensions().entrySet()) {
+                final String k = dimension.getKey();
+                if (!k.equals(CombinedMetricData.CLUSTER_KEY)
```

Comment:
Not sure what you mean sorry - as in, change "cluster" to CombinedMetricData.CLUSTER_KEY ? Or do you mean just allow the cluster dimension to be passed, and remove line 69?